### ISSUE TITLE
3.4.0 Keeps Blocking PlayStore

### ISSUE THREAD

**Issue by 0pLuS0** [ukanth/afwall#1123]
**Describe the bug**
AFWall+ keeps blocking the PlayStore, even though I have it enabled for Wifi and Mobile Data. If I disable and re-enable AFWall+ the PlayStore will work again.

**Firewall Logs**
Please get the log from Menu -> Show Rules -> (menu) Export to storage and attach it here

[IPv4rules.log](https://github.com/ukanth/afwall/files/4950299/IPv4rules.log)

**Smartphone (please complete the following information):**
 - Device: OnePlus 5T
 - Android OS: OxygenOS 9.0.11

**Additional context**
If this is a bug, I hope this issue will be resolved soon, if it's not a bug, then what is wrong?

THANKS

**Comment by 0pLuS0** [comment]
I was also having problems with connecting while using Chrome, it wouldn't load any sites until I disabled and enabled it.

AFWall+ Is not working good for me in OOS Pie, I never had these problems with it in OOS Oreo, I'm not going to use it now, until issues like these have been worked out.

**Comment by freebrowser1** [comment]
Weird.
I have an 1+6 with Oxygen OS 10.4 (stock) with Magisk root.
Play store works well, I have different profiles, one which blocks everything except browsers and email, and the other also allowing Google Play. The latter just works.

**Comment by ukanth** [comment]
I don't see a problem. @0pLuS0, next time if this issue happens, can you please capture the log before & after ? This could be due to active rules not working properly. Also can you please do a clean install ? Because I see lot of preferences not enabled by default.

**Comment by 0pLuS0** [comment]
@ukanth 

The first time I started using AFWall was in OxygenOS Oreo on the same device, what I did was only click on the apps I wanted to allow, everything else in the system including system apps that AFWall lists were not checked, and everything worked.

Now if I do the same thing in PIE, this is not working, Chrome sometimes would not connect, also the PlayStore, WiFi calling not working.

So how I only allowed a few apps, is this ok to still do in PIE?

I think for now I'll wait till the next version comes out, I've never had any of these issue in Oreo.

THANKS
